extensions/functions_arithmetic.yaml

```
@@ -173,6 +173,178 @@ scalar_functions:
+      - args:
+          - value: fp32
+        return: fp64
+      - args:
+          - value: fp64
+        return: fp64
+  -
+    name: "sin"
+    description: "Get the sine of a value in radians."
+    impls:
+      - args:
+          - value: i8
+        return: fp64
+      - args:
+          - value: i16
+        return: fp64
+      - args:
+          - value: i32
+        return: fp64
+      - args:
+          - value: i64
+        return: fp64
+      - args:
+          - value: fp32
+        return: fp64
+      - args:
+          - value: fp64
+        return: fp64
+  -
+    name: "tan"
+    description: "Get the tangent of a value in radians."
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i8
+        return: fp64
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i16
+        return: fp64
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i32
+        return: fp64
+      - args:
+         - options: [ SILENT, SATURATE, ERROR ]
+           required: false
+         - value: i64
+        return: fp64
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: fp32
+        return: fp64
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: fp64
+        return: fp64
+  -
+    name: "acos"
+    description: "Get the arc-cosine of a value in radians."
+    impls:
+      - args:
+          - value: i8
+        return: fp64
+      - args:
+          - value: i16
+        return: fp64
+      - args:
+          - value: i32
+        return: fp64
+      - args:
+          - value: i64
+        return: fp64
+      - args:
+          - value: fp32
+        return: fp64
+      - args:
+          - value: fp64
+        return: fp64
+  -
+    name: "asin"
+    description: "Get the arc-sine of a value in radians."
+    impls:
+      - args:
+          - value: i8
+        return: fp64
+      - args:
+          - value: i16
+        return: fp64
+      - args:
+          - value: i32
+        return: fp64
+      - args:
+          - value: i64
+        return: fp64
+      - args:
+          - value: fp32
+        return: fp64
+      - args:
+          - value: fp64
+        return: fp64
+  -
+    name: "atan"
+    description: "Get the arc-tangent of a value in radians."
+    impls:
+      - args:
+          - value: i8
+        return: fp64
+      - args:
+          - value: i16
+        return: fp64
+      - args:
+          - value: i32
+        return: fp64
+      - args:
+          - value: i64
+        return: fp64
+      - args:
+          - value: fp32
+        return: fp64
+      - args:
+          - value: fp64
+        return: fp64
+  -
+    name: "atan2"
+    description: "Get the arc-tangent of 2 values in radians."
```

```suggestion
    description: "Get the arctangent of values given as x/y pairs."
```
The inputs to atan2 are not actually in radians